templates/global/menu.html

```diff
@@ -15,6 +15,7 @@
                     <li><a href="{% url 'core:stories' %}">Stories</a></li>
                     <li><a href="{% url 'core:resources' %}">Resources</a></li>
                     <li><a href="{% url 'core:organize' %}">Organize</a></li>
+                    <li><a href="{% url 'jobs:jobs' %}">Jobs</a></li>
```

Comment:
For now, I would leave the jobs link out of the menu.  We can still get to it by going to djangogirls.org/jobs directly, and if we start to push code in stages to the main djangogirls repo we won't that link to be visible until the jobs part is ready